p2p/starknet/stream.go

```diff
@@ -0,0 +1,16 @@
+package starknet
+
+// Stream represents a series of messages that can be accessed by invoking Stream a number of times. With the last element in the stream
```

Comment:
Mind finishing off this comment? I'm genuinely interested in knowing what happens at the end :)